fastn-core/src/http.rs

```diff
@@ -91,6 +91,45 @@ pub fn ok_with_content_type(
         .body(data)
 }
 
+#[derive(Debug, Default, Clone)]
+pub enum LogLevel {
+    #[default]
+    Info,
+    Warning,
+    Error,
+}
```

Comment:
Nopes, lets be more logical. This is "generic logging". By logical we mean "precise logging" or whatever. Like we should have an exact enum of all log messages we can emit.

Reply:
ekind-okind decide if something is info warning etc. Also error is captured as outcome, and error details have to go to outcome-data etc.